.github/actions/high-priority-prs/process.js

```diff
@@ -151,11 +156,14 @@ module.exports = data => {
       _.maxBy(pr.comments.nodes, n => n.createdAt),
       `createdAt`
     )
+
+    const lastCommitMessage = pr.commits.nodes[0].commit.message
     const lastCommit = pr.commits.nodes[0].commit.authoredDate
```

Comment:
Here we check only last commit and this can cause some false negatives in situation like this:

```
 - last comment
 - meaningful commit
 - merge commit
```
I think we need to iterate commit list until we find non-merge commit and use date of that commit

Reply:
That's true. 
I will fix it soon.
(sorry... My english is not very good, but I'll try my best! )

Reply:
Your english is great! And thanks for addressing my comment!